keyboards/divergetm2/config.h

```diff
@@ -0,0 +1,188 @@
+/* Copyright 2018 Christon DeWan (xton)
+ * Copyright 2017 IslandMan93
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x1256
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    UniKeyboard
+#define PRODUCT         diverge tm2
+#define DESCRIPTION     Split 46 key keyboard
+
+/* key matrix size */
+#define MATRIX_ROWS 8
+#define MATRIX_COLS 6
+// SERIAL is the only supported
+#define USE_SERIAL
```

Comment:
This actually doesn't need to be defined.  The Split code should default to serial if nothing is defined.